andela/bp-esa-frontend
# #168094512 Remove retry button on successful automations

**What does this PR do?**
THis PR removes the retry button on details for successful automations

**Description of the task to be completed?**
- When a user clicks to view details of a successful automation, the retry button should mot appear as there is no need to rerun the automation.

**How should this be manually tested?**
- Clone and set up the project using the instruction in the README.
- $ git checkout to `ft-retry-button-on-details-168094512` branch which contains the changes.
- Use $ yarn start to start the application
- Start the frontend and Log in to the application.
- Click on automation details for a failed automation
The retry button should be existent on the automation details modal.
- Click on automation details for a successful automation
The retry button should not be existent on the automation details modal.

**What are the relevant pivotal tracker stories?**
[#168094512](https://www.pivotaltracker.com/story/show/168094512)

**Screenshots**
<img width="594" alt="Screenshot 2019-08-28 at 17 38 08" src="https://user-images.githubusercontent.com/38077368/63865667-a290a800-c9ba-11e9-9816-59b1fc880fee.png">
